Sourcery/Generating/Generator.swift

```diff
@@ -25,7 +25,7 @@ private class TypesReflectionBox: NSObject {
     lazy var classes: [Type] = {
         return self.types.filter { type in
             let isNotClass = type is Struct || type is Enum || type is Protocol
-            return !isNotClass
+            return !isNotClass && !type.isExtension
```

Comment:
ah right I was thinking this won't be consistent but we have fake types for all other kind of entries, so this is the only one we need to filter :+1: 

It will also be accessible from `types.based.ProtocolName` right? 
also :+1: on adding `extension` kind